homework/src/index.js

```diff
@@ -1,47 +1,119 @@
-'use strict';
-
+"use strict";
 {
-  function fetchJSON(url, cb) {
-    const xhr = new XMLHttpRequest();
-    xhr.open('GET', url);
-    xhr.responseType = 'json';
-    xhr.onload = () => {
-      if (xhr.status < 400) {
-        cb(null, xhr.response);
-      } else {
-        cb(new Error(`Network error: ${xhr.status} - ${xhr.statusText}`));
+  function main() {
+    fetchJSON("https://api.github.com/orgs/HackYourFuture/repos", (err, data) => {
```

Comment:
I'm seeing a subset of HYF repositories only. For instance, the JavaScript2 and JavaScript3 repos are not listed in your `select` element. This is because you left out the `perPage` query parameter that was explicitly mentioned in the assignment:

```
https://api.github.com/orgs/HackYourFuture/repos?per_page=100
```

By leaving it out, GitHub just returns the first 30 repos.